chainerx_cc/chainerx/native/kernel_regist.h

```diff
@@ -1,6 +1,12 @@
 #pragma once
 
+#include <cstdint>
+
+#include "chainerx/array.h"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
 #include "chainerx/kernel_registry.h"
+#include "chainerx/native/elementwise.h"
```

Comment:
Could you apply the same "fix" for `cuda/kernel_regist.h` and then fix the includes in `chainerx_cc/chainerx/cuda/cuda_device/rounding.cu` ?

Reply:
We should ideally go through the includes in the other files where `kernel_regist.h` have been included..

Reply:
Fixed `cuda/kernel_regist.h` except `elementwise.cuh` which cannot be included from `.h`.

Reply:
I see, thanks for the fix and the comment.